src/pages/_app.tsx

```diff
@@ -68,16 +68,20 @@ const networks = getSupportedNetworks().filter(chain =>
     chainIds.includes(chain.id)
 );
 
-const { chains, publicClient } = configureChains(networks, [
-    alchemyProvider({
-        apiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY ?? '',
-    }),
-    publicProvider(),
-]);
+const { chains, publicClient, webSocketPublicClient } = configureChains(
+    networks,
+    [
+        alchemyProvider({
+            apiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY ?? '',
+        }),
+        publicProvider(),
+    ]
+);
 
 const config = createConfig({
     autoConnect: false,
     publicClient: publicClient,
+    webSocketPublicClient,
```

Comment:
why do we need this?

Reply:
Added that for testing something previously. Removing it.